src/modules/Builder/features/ActivityItems/ItemConfiguration/ItemConfiguration.tsx

```diff
@@ -34,14 +34,17 @@ export const ItemConfiguration = ({ name, onClose }: ItemConfigurationProps) =>
   const [isEditItemPopupVisible, setIsEditItemPopupVisible] = useState(false);
   const selectChangeRef = useRef<undefined | (() => void)>();
 
-  const methods = useFormContext();
+  const { control } = useFormContext();
   const { fieldName, activity } = useCurrentActivity();
   const { message, isPopupVisible, onPopupConfirm } = useCheckIfItemHasVariables(name);
 
-  const { control, watch } = methods;
-  const isReviewable = watch(`${fieldName}.isReviewable`);
-  const responseType = watch(`${name}.responseType`) as ItemResponseTypeNoPerfTasks;
-  const currentItem = watch(name);
+  const [isReviewable, responseType, currentItem]: [
```

Comment:
pls move types to ItemConfiguration.types.ts

Reply:
done